src/script/common/c_content.cpp

```diff
@@ -1067,30 +1076,30 @@ NodeBox read_nodebox(lua_State *L, int index)
 	}
 
 #define NODEBOXREADVEC(n, s) \
-		lua_getfield(L, index, (s)); \
-		if (lua_istable(L, -1)) \
-			(n) = read_aabb3f_vector(L, -1, BS); \
-		lua_pop(L, 1);
+	lua_getfield(L, index, (s)); \
+	if (lua_istable(L, -1)) \
+		(n) = read_aabb3f_vector(L, -1, BS); \
+	lua_pop(L, 1);
```

Comment:
Prime candidate for replacing with a lambda

Reply:
Indeed, although it doesn't quite fit into the scope/goals of this PR. The lines were only re-indented.

Reply:
Right.